### PR TITLE
Serilog & app insights configuration

### PR DESCRIPTION
**Pull request by timabell**

### CODE REVIEW

**Comment by timabell** [comment]
[Add Azure-friendly trace logging for 4x microservices + webjob](https://trello.com/c/mSf3x01B/158-add-azure-friendly-trace-logging-for-4x-microservices-webjob)

**Comment by timabell** [comment]
[Add Azure-friendly trace logging for 4x microservices + webjob](https://trello.com/c/mSf3x01B/158-add-azure-friendly-trace-logging-for-4x-microservices-webjob)

File: src/Program.cs [hunk 1]
```diff
@@ -1,25 +1,60 @@
...
+            finally
+            {
+                Log.CloseAndFlush();
+            }
         }
 
-        public static IWebHostBuilder CreateWebHostBuilder(string[] args) =>
+        public static IWebHost BuildWebHost(string[] args) =>
             WebHost.CreateDefaultBuilder(args)
-                .UseApplicationInsights()
```
Comment (MartinBentley) [inline src/Program.cs]: Do we not need this line?

Comment (timabell) [reply src/Program.cs]: doh

**Pull request merged**